shared/src/userData.ts

```diff
@@ -73,6 +80,7 @@ export const createEmptyBusiness = (id?: string): Business => {
     },
     environmentData: undefined,
     version: CURRENT_VERSION,
```

Comment:
@somabadri I just thought about this, but we should probably also have `versionWhenCreated` as well.

This would be a natural time to add this to the object. Sorry I didn't think about this earlier.